live/mirage/models/organization.js

```diff
@@ -0,0 +1,4 @@
+import { Model } from 'ember-cli-mirage';
+
+export default Model.extend({
```

Comment:
Mirage needs this model when creating and reading organization data. But we are not forced to define fields to this model.